specs/agents/tracing-instrumentation-db.md

```diff
@@ -173,7 +173,7 @@ If they don't, they MUST use a hard-coded list of URLs that correspond with the
 |`action`|e.g. `find` , `insert`, etc.| The MongoDB command executed with this action. |
 | __**context.db._**__  |<hr/>|<hr/>|
 |`_.instance`| e.g. `customers` | Database name, if available |
-|`_.statement`| e.g. <pre lang="json">find({status: {$in: ["A","D"]}})</pre> | The MongoDB command encoded as MongoDB Extended JSON.|
+|`_.statement`| e.g. <pre lang="json">find({status: {$in: ["A","D"]}})</pre> | The MongoDB command encoded as MongoDB Extended JSON, if `mongodb_capture_statement` is set to `true` |
```

Comment:
Just a thought, but similar to `elasticsearch_capture_body_urls`, we could also have a configurable list of MongoDB commands where we capture the full statement. That would let users capture `find, count` but not `insert`, for example.

Reply:
I think it would be a great follow-up task on this PR, but likely better handled in a separate PR, first with a hardcoded list of commands when agents have when `mongo_capture_statement=true`, then an optional configuration option that might allow to customize it if required. For example the similar Elasticsearch configuration option is only implemented in NodeJS agent as far as I know.

Also, it would allow us to properly implement what the documentation says about the DB statement capture:
https://www.elastic.co/guide/en/apm/guide/current/filtering.html#filters-database-statements.

Reply:
Or, maybe better, we could add the command exclusion part (at least the hardcoded variant) directly into this PR.

Reply:
Thanks for that link. I hadn't been aware of that claim to not capture the body for inserts and updates for MongoDB. +1 to considering a config var that is more than a boolean to be able to capture query-type MongoDB statements by default, but not inserts/updates.